xla/service/gpu/buffer_comparator_test.cc

```diff
@@ -52,6 +52,13 @@ class BufferComparatorTest : public testing::Test {
         stream_exec_(platform_->ExecutorForDevice(0).value()) {
   }
 
+  const DebugOptions& debug_options() const { 
```

Comment:
It looks to me these two methods are unused?

Reply:
indeed.. after the last small refactoring, module config is no longer needed, will take care of this today